### PR TITLE
US-3.4: Add perkstore and wallet portlets

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/perkstore.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="VuetifyApp">
+    <div id="digital-workplace-perkstore">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/perkstore.html]: We should call init function here, see https://github.com/exo-addons/digital-workplace/commit/b974b4d41f72299251803e179042452f53c0fbe1

File: digital-workplace-webapps/src/main/webapp/vue-app/perkstore/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    iconfont: '',
+});
+
+// getting language of user
+const lang = eXo && eXo.env && eXo.env.portal && eXo.env.portal.language || 'en';
+
+const resourceBundleName = 'locale.addon.DigitalWorkplace';
+const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/${resourceBundleName}-${lang}.json`;
+
+// getting locale ressources
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/perkstore/main.js]: We should add init function here, see https://github.com/exo-addons/digital-workplace/commit/b974b4d41f72299251803e179042452f53c0fbe1

File: digital-workplace-webapps/src/main/webapp/vue-app/profile-stats/components/UserProfile.vue [hunk 3]
```diff
@@ -25,123 +25,7 @@
...
-                        </v-card>
-                      </v-flex>
-                    </v-layout>
-                  </v-flex>
-                </v-layout>
-              </v-flex>
-            </v-layout>
-          </v-flex>
-        </v-layout>
+
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/profile-stats/components/UserProfile.vue]: You should rename this file ProfileStats.vue, and rename ProfileStats.vue to UserDashbord.vue

File: digital-workplace-webapps/src/main/webapp/vue-app/wallet/main.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    iconfont: '',
+});
+
+// getting language of user
+const lang = eXo && eXo.env && eXo.env.portal && eXo.env.portal.language || 'en';
+
+const resourceBundleName = 'locale.addon.DigitalWorkplace';
+const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/${resourceBundleName}-${lang}.json`;
+
+// getting locale ressources
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/wallet/main.js]: Same remark of Perkstore portlet

File: digital-workplace-webapps/src/main/webapp/wallet.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<div class="VuetifyApp">
+    <div id="digital-workplace-wallet">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/wallet.html]: Same remark of Perkstore portlet

Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/wallet.html]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/profile-stats/components/UserProfile.vue (hunk 3 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/profile-stats/components/UserProfile.vue]: Done!

**Pull request merged**